Makefile

```diff
@@ -19,11 +17,11 @@ tmp:
 
 # Run the test suite, optionally with coverage
 test: tmp
-	pytest tests/unit -c tests/unit/pytest.ini
+	pytest tests/unit
```

Comment:
Just a suggestion - these can be hatch environment scripts, tmt [here](https://github.com/teemtee/tmt/blob/e5eabda6a9a2875e92a88b6bd2a6b9eb034fd748/pyproject.toml#L134) and [here](https://github.com/teemtee/tmt/blob/e5eabda6a9a2875e92a88b6bd2a6b9eb034fd748/Makefile#L26). 
The advantages are having clean reproducible test environments with dependency control.

Reply:
I am not familiar with `hatch` environments. In the makefile I think it is helpful since the appropriate `pip install -e .[test]` might not have been run. But at the same time, I am not a fan of have manual makefiles either. Could you help set that up later?

For dependency control, I have a different opinion that I will express in the other comment.